### ISSUE TITLE
Investigate if we can automate pending rulemaking wagtail page

### ISSUE THREAD

**Issue by rfultz** [fecgov/fec-cms#6580]
### Summary

**What we're after:**
We want to utilize Wagtail CMS’s scheduled publishing and unpublishing capabilities for the [Pending Rulemakings page](https://www.fec.gov/legal-resources/regulations/pending-rulemaking-matters-comment/). This functionality will ensure timely updates by automating page visibility based on predefined dates and times, similar to the scheduling process for homepage banners.

#### Example of pending rulemaking page

![Image](https://github.com/user-attachments/assets/be07d107-5d4e-4f90-9db7-923d9d89f5d3)

#### Corresponding homepage banner for pending rulemaking comments

![Image](https://github.com/user-attachments/assets/b7a5aa0e-96f6-4c4c-bdcb-ae4624b3fc7c)

### Related issues

- #6554 

### Action items
- A documented plan for enabling scheduled publishing and unpublishing for the Pending Rulemakings page.
   - Investigate Wagtail CMS documentation and settings for scheduling publishing/unpublishing, in the page's status menu
   - Identify any dependencies or limitations specific to the Pending Rulemakings page.
- Clear steps for implementation, including any required configuration in Wagtail CMS.
- Confirmation that the scheduling works as expected, including:
   - Proper display of the page after scheduled publication.
   - Automatic removal of the page after the scheduled unpublishing.
- Testing results to confirm no disruptions to existing content workflows or the CMS.

### Completion criteria

- [ ] Scheduled publishing and unpublishing of the pending rulemaking page is implemented
- [ ] A documented plan is written in conjunction with the content team

**Comment by johnnyporkchops** [comment]
As of version  5.1, scheduled publishing is handled in the pages status menu. We need to test whether this works as expected and make sure we don't see the same issues with time zone mismatch with EST (America/New York) like we did in earlier Wagtail versions when the settings  tab had the `Go-live` and `Expiry` fields.
https://docs.wagtail.org/en/v5.1.3/releases/4.1.html#new-ui-for-scheduled-publishing

**Comment by patphongs** [comment]
Thanks @johnnyporkchops for your work on this. I have a question about what happens if there's multiple rulemakings that are listed with different opening and closing comment period dates? Would we have to edit the page and modify the listing separately?

**Comment by johnnyporkchops** [comment]
> Thanks [@johnnyporkchops](https://github.com/johnnyporkchops) for your work on this. I have a question about what happens if there's multiple rulemakings that are listed with different opening and closing comment period dates? Would we have to edit the page and modify the listing separately?

@patphongs No, the rulemaking CustomPage would not have to be manually edited. I demoed this scenario to the Content team  and it is probably better to demo it again so you can see, rather than attempt to explain it in writing.

**Comment by cnlucas** [comment]
Research has been logged in this ticket, ticket for implementation here
